sn_transfers/src/wallet/error.rs

```diff
@@ -37,7 +43,10 @@ pub enum Error {
     FailedToHexEncodeKey(String),
     /// Dbc error.
     #[error("Dbc error: {0}")]
-    Dbc(#[from] Box<sn_dbc::Error>),
+    BoxedDbc(#[from] Box<sn_dbc::Error>),
+    /// Dbc error.
+    #[error("Dbc error: {0}")]
+    Dbc(#[from] sn_dbc::Error),
```

Comment:
Why do we need 2 different error types here?

Reply:
refactored away 👍